tracing-log/src/lib.rs

```diff
@@ -9,6 +28,12 @@
 //! records emitted by dependencies which use `log` within the context of a
 //! trace.
 //!
+//! ## Convert tracing `Event`s to logs
+//!
+//! This conversion can be done with [`TraceLogger`].
```

Comment:
What about maybe:
```rust
//! This conversion can be done with [`TraceLogger`], a [`Subscriber`] which 
//! records `tracing` spans and events and outputs log records.
```
or something like that?

Reply:
:+1: